src/main/resources/io/cryostat/net/openshift/OpenShiftAuthManager.properties

```diff
@@ -8,7 +8,7 @@
 # RECORDINGS="recordings.operator.cryostat.io"
 
 # ResourceType to Kubernetes resource kind mapping:
-TARGET=flightrecorders.operator.cryostat.io
-RECORDING=recordings.operator.cryostat.io
+TARGET=pods,deployments.apps
```

Comment:
For targets, I wonder if `endpoints` is also necessary? My thinking is that discovery mechanism needs access endpoint resource so it should be declared here.

Reply:
Makes sense to me.

Reply:
I just stumbled across this note that write access to Endpoints objects is going to be forbidden in most scenarios in Kubernetes 1.22+: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#write-access-for-endpoints

This may make `CREATE`, `UPDATE`, `DELETE` operations on targets difficult if we include Endpoints.

Reply:
Fair enough. One way around would be to differentiate between normally discovered targets (readonly) and custom targets (all verbs), but that feels like a bit of a hack.

Reply:
For that security issue, I think we could remove it. I was looking into a replacement but I wonder if a more "
abstract" object like `service` is a good alternative?

Reply:
Service seems like a reasonable substitute since they're so closely coupled.

Reply:
These are all somewhat arbitrarily chosen, it would be good to allow users to configure this permissions mapping file from the operator. I'm not sure if we have an existing issue for that.

Reply:
Hmm I cant see one but I only remember Andrew mentioned it in this issue https://github.com/cryostatio/cryostat/issues/979#issuecomment-1146325376. In this case, we will allow some config in Cryostat for this mapping, right?

Reply:
The file you edited in this PR is that mapping file, see #662 . There may be a bit more work required to make this something that the Operator can replace at runtime, but the idea is that the Operator could just mount a replacement file overtop of this default one in the container image.

Reply:
I filed an issue in the operator for this feature: https://github.com/cryostatio/cryostat-operator/issues/439

Reply:
Nicee! I will dig more into this auth flow and look at the new issue too!